xml/System.Windows.Forms/SystemInformation.xml

```diff
@@ -1648,13 +1638,8 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Windows.Forms.SystemInformation.IsActiveWindowTrackingEnabled%2A> property indicates whether active window tracking is enabled. Active window tracking causes any window the mouse is directly over to become the active window.  
-  
-> [!NOTE]
->  This property is supported only on Windows 98, Windows Millennium Edition, Windows 2000, Windows XP, and the Windows Server 2003 family.  
```

Comment:
Supported on W2000+

Reply:
Are you saying that we should mention the minimum OS that supports this?

It doesn't seem to be necessary anymore: we currently support this API in all the operating systems where .NET can run, so there's no point in mentioning older unsupported OS.

Reply:
That was the source of my confusion. Since we're only targeting .NET-supported OS's all these notes can be removed.